jsinterop-ts-defs-processor/src/test/resources/types/test.ts

```diff
@@ -698,6 +706,18 @@ enumClient.values = {
     test3: [1, 2, 3],
 }
 
+DayOfWeek.values();
+class ItemDetailsChild extends ItemDetails {
+    constructor() {
+        super();
+    }
+}
+const itemDetails = new ItemDetailsChild();
+// TODO : The dtslint will always unwrap the type alias, will need to look for a different way to test this.
```

Comment:
yeah, we need proper type union/aliasing, this is definitely the wrong way to do it... not today though.